README.md

```diff
@@ -1 +1,37 @@
-# andre-acosta-portfolio
\ No newline at end of file
+Andre Acosta Portfolio
```

Comment:
- [ ] To keep your repo professional, you should consider using [Microverse's template](https://github.com/microverseinc/readme-template) instead of only plain text. You can read more about `.md` syntax [here](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).